redaxo/src/core/tests/sql/sql_table_test.php

```diff
@@ -163,6 +164,60 @@ public function testAddColumn()
         static::assertSame(['pid', 'id', 'name', 'title', 'description'], array_keys($table->getColumns()));
     }
 
+    public function testAddColumnComment()
+    {
+        $table = $this->createTable();
+
+        $title = new rex_sql_column('title', 'varchar(20)', false, null, null, 'new title comment');
+        $table
+            ->ensureColumn($title)
+            ->alter();
+
+        static::assertSame($title, $table->getColumn('title'));
+
+        rex_sql_table::clearInstance(self::TABLE);
+        $table = rex_sql_table::get(self::TABLE);
+
+        static::assertEquals($title, $table->getColumn('title'));
+        static::assertSame('new title comment', $table->getColumn('title')->getComment());
+    }
+
+    public function testChangeColumnComment()
+    {
+        $table = $this->createTable();
+
+        $id = new rex_sql_column('id', 'int(11)', false, null, 'auto_increment', 'changed id comment');
+        $table
+            ->ensureColumn($id)
+            ->alter();
+
+        static::assertSame($id, $table->getColumn('id'));
+
+        rex_sql_table::clearInstance(self::TABLE);
+        $table = rex_sql_table::get(self::TABLE);
+
+        static::assertEquals($id, $table->getColumn('id'));
+        static::assertSame('changed id comment', $table->getColumn('id')->getComment());
+    }
+
+    public function testRemoveColumnComment()
+    {
+        $table = $this->createTable();
+
+        $id = new rex_sql_column('id', 'int(11)', false, null, 'auto_increment', null);
+        $table
+            ->ensureColumn($id)
+            ->alter();
+
+        static::assertSame($id, $table->getColumn('id'));
+
+        rex_sql_table::clearInstance(self::TABLE);
+        $table = rex_sql_table::get(self::TABLE);
+
+        static::assertEquals($id, $table->getColumn('id'));
+        static::assertNull($table->getColumn('id')->getComment());
```

Comment:
Mich wundert dass der letzte kleine fehler nicht via psalm/phpstan aufgefallen ist

@gharlan gibt es ein psalm plugin für phpunit? Sodass die asserts schon zu analyste zeit fehltschlagen?

Reply:
ich glaube wir haben die tests insgesamt in psalm noch nicht drin, du hast sie nur irgendwann zu phpstan hinzugefügt.

Reply:
Gibt aber ein Plugin: https://github.com/psalm/psalm-plugin-phpunit

Reply:
Here we go

https://github.com/redaxo/redaxo/pull/3715

https://github.com/redaxo/redaxo/pull/3716